src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+### Fund your ideas {#crowdfunding}
+
+Ethereum is open source software. A lot of the work so far has been funded by the community and this has led to the growth of an interesting new fundraising model: quadratic funding. This method has the potential to improve the way we fund all types of public goods in the future.
+
+Quadratic funding makes sure that the projects that receive the most funding are those with the most unique demand. In other words, projects that stand to improve the lives of the most people. Here's how it works:
+
+1. There is a matching pool of funds donated.
+2. A round of public funding starts.
+3. People can signal their demand for a project by donating some money.
+4. Once the round is over, the matching pool is distributed to projects. Those with the most unique demand get the highest amount from the matching pool.
+
+This means project A that received 100 donations of 1 dollar could end up with more funding than project B which received a single donation of 10,000 dollars (dependent on the size of the matching pool).
+
+## How does defi work?
+
+Defi uses cryptocurrencies and smart contracts to provide services that don't need intermediaries. In today's financial world, financial institutions acts as guarantors of transactions. This gives these institutions immense power – even the power to stop you making transactions because your money flows through them. And discrimination is rife – billions of people around the world don't have access to a bank account.
+
+In defi, a smart contract replaces the financial institution in the transaction. A smart contract is a type of Ethereum account that can hold funds and can send/refund them based on certain conditions. No one can alter that smart contract when it's live. So it will always run exactly as programmed.
+
+A pocket money contract that's programmed to send money from Account A to Account B every Friday will only ever do that. So long as Account A has the required funds. No one can change the contract and add Account C as a recipient to steal funds.
+
+## Ethereum and defi
+
+Ethereum is the perfect foundation for defi for a number of reasons:
+
+- No one owns Ethereum or the smart contracts that live on it – this gives everyone an opportunity to use defi.
+- Defi products all speak the same language behind the scenes: Ethereum. This means many of the products can be used in tandem. You can lend tokens on one platform, and exchange the interest-bearing token in a different market on an entirely different application. This is like being able to cash your Starbucks loyalty points in at your bank.
```

Comment:
"can be used in tandem" not sure if that's clear. Maybe "are interoperable"? Or "many of the products can be easily integrated" / "work together seamlessly"?